practice1/stylesheet.css

```diff
@@ -75,15 +75,75 @@ hr {
 }
 .blocks-layout {
     display: grid;
-    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
     grid-template-columns: 1fr 1fr 1fr;
     grid-gap: 1vw;
+    margin-bottom: 62px;
+}
+.shop-btn-container {
+    display: flex;
+    align-items: center;
+    gap: 8px;
 }
 .img-text {
     position: relative;
 }
 .img-text h4 {
-    position: absolute;
     margin-top: 12px;
     margin-left: 12px;
+}
+.caption-favorite {
+    display: flex;
+    position: absolute;
+    width: 450px;
+    justify-content: space-between;
+}
+.caption-favorite img {
+    align-self: center;
+    margin-right: 15px;
+    image-resolution: snap;
```

Comment:
<img width="275" alt="image" src="https://user-images.githubusercontent.com/16446380/194163969-e25f8fbf-a0f6-41f7-8438-ff82310ba89f.png">
My Chrome doesn't know what this is 😟